src/components/universalSearch/searchBar/stores/searchBarUiStateStore.js

```diff
@@ -88,7 +88,8 @@ export class SearchBarUiStateStore {
         // construct current search object using observables
         const search = SearchBarUiStateStore.turnChipsIntoSearch(this.chips);
 
-        const showAllTabs = Object.keys(search).every((key) => key === 'serviceName' || key === 'operationName');
+        const showAllTabs = Object.keys(search).every((key) => key === 'serviceName' || key === 'operationName') || this.tabId === 'serviceInsights';
```

Comment:
Adds a `|| this.tabId === 'serviceInsights` so that if a user is accessing service insights, we do not navigate them away from the feature no matter the combination of search inputs in the search box.  (Previously, entering a traceId here would navigate them to traces)

CC @nathanwalther